### PR TITLE
chore: improve QA

### PR DESCRIPTION
**Pull request by pedroferreira1**
[Rendered](https://github.com/HathorNetwork/hathor-wallet-headless/blob/chore/improve-qa/QA.md)

### Acceptance criteria

- The QA should have all operations available in the wallet headless

### CODE REVIEW

File: QA.md [hunk 1]
```diff
@@ -1,31 +1,43 @@
...
+    1. Now we will mint more tokens with `curl -X POST -H "X-Wallet-Id: 123" -H "Content-type: application/json" --data '{"token": <TOKEN_UID>, "amount": 100}' http://localhost:8000/wallet/mint-tokens`. Now check the balance with `curl -X GET -H "X-Wallet-Id: 123" http://localhost:8000/wallet/balance?token=<TOKEN_UID>` the balance must be 200.
+    1. Now we will melt tokens with `curl -X POST -H "X-Wallet-Id: 123" -H "Content-type: application/json" --data '{"token": <TOKEN_UID>, "amount": 100}' http://localhost:8000/wallet/melt-tokens`. Now check the balance with `curl -X GET -H "X-Wallet-Id: 123" http://localhost:8000/wallet/balance?token=<TOKEN_UID>` the balance must be 100.
+    1. Now we will create an NFT. `curl -X POST -H "X-Wallet-Id: 123" -H "Content-type: application/json" --data '{"name": "Test Token", "symbol": "TST", "amount": 1, "data": "nft data", "address": <first_address>}' http://localhost:8000/wallet/create-nft`.
```
Comment (andreabadesso) [inline QA.md:43]: What should we expect? 200 status code? Something on the body?

**Review by andreabadesso — CHANGES_REQUESTED** [review]

File: QA.md (hunk 1 above)
Comment (pedroferreira1) [reply QA.md]: Added here https://github.com/HathorNetwork/hathor-wallet-headless/pull/124/commits/18e178437553c59b40d6716344b67b14df647aa5

**Review by andreabadesso — APPROVED** [review]

File: QA.md [hunk 2]
```diff
@@ -1,31 +1,43 @@
...
+    1. Now we will mint more tokens with `curl -X POST -H "X-Wallet-Id: 123" -H "Content-type: application/json" --data '{"token": <TOKEN_UID>, "amount": 100}' http://localhost:8000/wallet/mint-tokens`. Now check the balance with `curl -X GET -H "X-Wallet-Id: 123" http://localhost:8000/wallet/balance?token=<TOKEN_UID>` the balance must be 200.
+    1. Now we will melt tokens with `curl -X POST -H "X-Wallet-Id: 123" -H "Content-type: application/json" --data '{"token": <TOKEN_UID>, "amount": 100}' http://localhost:8000/wallet/melt-tokens`. Now check the balance with `curl -X GET -H "X-Wallet-Id: 123" http://localhost:8000/wallet/balance?token=<TOKEN_UID>` the balance must be 100.
+    1. Now we will create an NFT. `curl -X POST -H "X-Wallet-Id: 123" -H "Content-type: application/json" --data '{"name": "Test Token", "symbol": "TST", "amount": 1, "data": "nft data", "address": <first_address>}' http://localhost:8000/wallet/create-nft`. The response must be success true and the created transactions data.
```
Comment (luislhl) [inline QA.md:43]: Should we add this `/create-nft` api in the README.md?

**Review by luislhl — APPROVED** [review]

File: QA.md (hunk 2 above)
Comment (pedroferreira1) [reply QA.md:43]: Created an issue for that https://github.com/HathorNetwork/hathor-wallet-headless/issues/125

**Pull request merged**